scala2java-test-utils/src/main/scala/io/github/effiban/scala2java/test/utils/integration/matchers/FileMismatchMessageGenerator.scala

```diff
@@ -24,8 +24,8 @@ private[matchers] object FileMismatchMessageGenerator {
   private def generate(expectedLines: Seq[String], actualLines: Seq[String], mismatchingLineNum: Int): String = {
     val maxNumLines = max(actualLines.size, expectedLines.size)
     val maxLineNumDigits = numDigitsOf(maxNumLines)
-    val maxExpectedLineLength = expectedLines.map(_.length).max
-    val maxActualLineLength = actualLines.map(_.length).max
+    val maxExpectedLineLength = maxOrZero(expectedLines)
+    val maxActualLineLength = maxOrZero(actualLines)
```

Comment:
This fixes a bug in case the sequence is empty since `max` cannot be applied to that